src/Webservices/Profile.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Profile class.
+ *
+ * Profiles are like schemas and define a kind of template which is used to construct a resource
+ * object in memory.  Representations can then be derived from the resource object.  A profile
+ * defines the name and type of each property and could be extended with other information.
+ * An automatic process could be written to generate machine-readable profile documents, such as
+ * ALPS or JSON-LD contexts from the profile.
+ *
+ * New data types can be added by adding a class file in the Type directory.  The new type can
+ * then be referenced in the profile.
+ */
+
+namespace Joomla\Webservices\Webservices;
+
+class Profile
```

Comment:
It makes a lot of sense - i really hate the `ConfigurationHelper` class at the moment - it mushes all kinds of horrible crap together - but I dunno if it's a necessary evil or not at the moment. But for the sake of simplicity maybe it's simpler to leave this out at the moment if it's not working (the small the amount of changes the easier code reviews + bug fixes are :P)